src/lib/decks.ts

```diff
@@ -6,7 +6,7 @@ import hospitalJson from '$lib/crc-decks/hospital.json';
 
 export const withId: <T extends object>(o: T) => Keyed<T> = (function () {
   let nextId = 0;
-  return (o) => ({ ...o, id: nextId++ });
+  return (o) => ({ id: nextId++, ...o });
```

Comment:
This change allows existing ids to persist, even if they will collide (ie `o.id > nextId`). I'm ok with persistence as long as we guarantee unique ids!

Reply:
The collision of generated IDs is interesting. Added a "skip this" set for this.

I think if we want to in general guarantee 0 collisions, we would need a single store that would become basically a KV for heterogeneous objects (id => CardProps). Id assignment (this logic) would be done on insertion. ... After sitting on this for 5 minutes, this seems reasonable but:
- it would require some code crunching to bring the rest of the app to use it
- it feels somewhat un-svelte-like since we'd no longer be passing state around with props

Reply:
Hmm, I think it's worthwhile as we are also tracking vcs/llm changes with IDs. If we were only using it for rendering it wouldnt be such an issue.